src/sustainable-web-design.js

```diff
@@ -331,6 +366,24 @@ class SustainableWebDesign {
       productionEnergy: formatNumber(annualEnergy * PRODUCTION_ENERGY),
     };
   }
+
+  ratingScale(co2e) {
+    if (lessThanEqualTo(co2e, 0.095)) {
```

Comment:
I think the documentation explaining the reasoning for these number is described here, and when I first looked over this code, these numbers felt a bit magical. These are percentiles, so the 5th percentile refers to values in the bottom 5% of the distribution, the 10th percentile the bottom 10% and so on, and we don't expect these to change.

For values we don't expect to change, we've used CONSTANTS before in this codebase, and when debugging, I can see scenarios where seeing knowing where TENTH_PERCENTILE would be more meaningful to our future selves than `0.186`.

How would you feel about and approach along these lines, where we use constants as a form of documentation for the key `ratingScale` function?

```js
FIFTH_PERCENTILE = 0.095 // in the bottom 5% 
TENTH_PERCENTILE = 0.186 // in the bottom 10% 

// values from June 1 crawl of HTTP Archive. More below:
// https://sustainablewebdesign.org/digital-carbon-ratings/#h-a-digital-carbon-rating-scale


// abridged code example
if (lessThanEqualTo(co2e, FIFTH_PERCENTILE)) {
  return "A+";
} else if (lessThanEqualTo(co2e, TENTH_PERCENTILE)) {
  return "A";
}
// and so on 
```

This makes the intention clearer, without needing to refer to the docs each time.